packages/astro/src/vite-plugin-build-html/index.ts

```diff
@@ -140,8 +137,7 @@ export function rollupPluginAstroBuildHTML(options: PluginOptions): VitePlugin {
           for (let node of findAssets(document)) {
             if (isBuildableLink(node, srcRoot)) {
               const href = getAttribute(node, 'href')!;
-              const linkId = viteifyPath(href);
-              assetImports.push(linkId);
+              assetImports.push(href);
```

Comment:
@matthewp since the change to the `viteifyURL()` function, I believe the `href`s are already correctly-formatted and Vite knows how to resolve them. Is this correct?